app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -0,0 +1,5 @@
+<p>Welcome <%= @email %>!</p>
+
```

Comment:
Maybe add sentence here that says something like "You have a new account for DPLA Primary Source Sets" so that people know what the account is for.